src/components/user-profile.html

```diff
@@ -183,7 +198,7 @@
           },
           localStorageClass: {
             type: Object,
-         },
+          },
```

Comment:
this one looks like you may need to remove all the spacing and re-tab. I'm not sure why it'd take a space to get it in the right place.

Reply:
that's weird I just reformatted that whole section in the last commit